src/main/java/net/dv8tion/jda/core/requests/restaction/RoleOrderAction.java

```diff
@@ -92,6 +95,11 @@ public Guild getGuild()
     @Override
     protected void finalizeData()
     {
+        final Member self = guild.getSelfMember();
+        final int selfPos = guild.getRoles().indexOf(self.getRoles().get(0));
```

Comment:
Also, I'm not quite sure why you're doing `indexOf` here. Use `.getRoles().get(0).getPosition()`